Dockerfile.wsgi

```diff
@@ -0,0 +1,5 @@
+ARG commit=latest
+FROM elifesciences/profiles_cli:${commit}
+
+ENV LANG=en_US.UTF-8
+ENTRYPOINT ["venv/bin/uwsgi", "--ini=/srv/profiles/uwsgi.ini"]
```

Comment:
By customizing the entrypoint and moving up the environment variable I suspect we could use just a single image rather than a separate one for the application server (`wsgi` here, `fpm` in the PHP case).